src/client/config.js

```diff
@@ -0,0 +1 @@
+export const API_URL = 'localhost:8080';
```

Comment:
Please remove if possible -- we can do this relative to the accessed URL

Reply:
I don't think we should couple anything back-end related to the front end